TestProjects/Lightmapping/Assets/Tests/Editor/Tests_Unit_Editmode/Editmode_BakeRestart_Tests.cs

```diff
@@ -278,4 +278,55 @@ public void GPULM_ChangeSamplesBetweenBakes_DoesNotFallbackToCPU()
 
         clearAll();
     }
+
+    [UnityTest] // Case 1364204
+    public IEnumerator DisablingRadeonProDenoiser_DuringABake_DoesNotFallbackToCPU()
```

Comment:
I would rather call this test:

RadeonProDenoiser_DisablingRadeonProDenoiserDuringABake_DoesNotFallbackToCPU

Reply:
Done, thanks for the suggestion